src/site/content/en/fast/your-first-performance-budget/index.md

```diff
@@ -89,7 +89,7 @@ Let's take a highly specialized search engine, doggos.io, as an example. Doggos.
 
 ## Competitive analysis
 
-Once you've analyzed your own site, it's time to analyze your competitors' sites. Comparing results from websites similar to yours is a great way to figure out a performance budget. Whether you are working on an established project or starting from scratch, this is an important step. You get competitive advantage when you are faster than your competitors.
+Once you've analyzed your own site, it's time to analyze your competitors sites. Comparing results from websites similar to yours is a great way to figure out a performance budget. Whether you are working on an established project or starting from scratch, this is an important step. You get competitive advantage when you are faster than your competitors.
```

Comment:
```suggestion
Once you've analyzed your own site, it's time to analyze your competitors' sites. Comparing results from websites similar to yours is a great way to figure out a performance budget. Whether you are working on an established project or starting from scratch, this is an important step. You get competitive advantage when you are faster than your competitors.
```

Reply:
https://examples.yourdictionary.com/examples-of-plural-possessive-nouns.html

Reply:
Reverted changes as per suggestions. Thanks!